code/Controllers/ContentController.php

```diff
@@ -297,9 +297,13 @@ public function getMenu($level = 1)
 
     /**
      * @return ArrayList<SiteTree>
+     * @deprecated 5.4.0 Use getMenu() instead. You can continue to use $Menu in templates.
      */
     public function Menu($level)
     {
+        Deprecation::withSuppressedNotice(
+            fn() => Deprecation::notice('5.4.0', 'Use getMenu() instead. You can continue to use $Menu in templates.')
+        );
         return $this->getMenu($level);
     }
```

Comment:
This method existed because of a limitation in the templating engine where if you use `$Menu(2)` for example, it would call `getMenu()` but not pass the args.

I've resolved that limitation, so it'll now pass the args through to the getter. So now we can get rid of this pointless method.